src/widgets.ts

```diff
@@ -83,6 +85,39 @@ export abstract class Widget<
     return dispatchWidgetPostMessageEvent
   }
 
+  /**
+   * Public method to communicate with the iframe widget about navigation events
+   * Can be called when the host has a 'back' event happen.
+   * This will send a post message event to the iframe widget, which if it's
+   * listening for the 'mx/navigation' event, it can make its changes and then
+   * send its own post message event back with a `did_go_back` properity.
+   */
+  navigateBack() {
```

Comment:
Can we add a type to this method? `navigateBack(): Promise<boolean>` should work.